liquibase/changeLogs/capture/functions/pruneTimeSeriesData/pruneTimeSeriesData.sql

```diff
@@ -0,0 +1,70 @@
+create or replace function ${AQTS_SCHEMA_NAME}.prune_time_series_data(date timestamp)
```

Comment:
I think we should working with a date here, not a timestamp. Date might be a reserved word, if so the name of the parameter should be something else. At the very least, line 1 here needs to match line 16 in the changelog ('character varying' vs 'timestamp')